utils/release_helper.py

```diff
@@ -0,0 +1,788 @@
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '-c', '--creds-file', default='~/secret.ini',
+        help="INI file containing Github token.")
+    parser.add_argument(
+        '-r', '--repository', action='store', default='ComplianceAsCode/content',
+        help="Project repository name.")
+
+    subparsers = parser.add_subparsers(dest='subcmd', required=True)
+    subparsers.add_parser(
+        'stats', help="Return information about the repository current state.")
+
+    cleanup_parser = subparsers.add_parser(
+        'cleanup', help="Cleanup after completing a release.")
+    cleanup_parser.add_argument(
+        '-b', '--branch', action='store_true',
+        help="Remove temporary branch used during stabilization phase.")
+
+    release_prep_parser = subparsers.add_parser(
+        'release_prep', help="Prepare the repository for the next release.")
+    release_prep_parser.add_argument(
+        '-c', '--contributors', action='store_true',
+        help="Update the contributors lists. Do it before creating the stabilization branch.")
+    release_prep_parser.add_argument(
+        '-b', '--branch', action='store_true',
+        help="Create the stabilization branch for the next release.")
+    release_prep_parser.add_argument(
+        '-m', '--milestone', action='store_true',
+        help="Create the next milestone and close the current milestone.")
+    release_prep_parser.add_argument(
+        '-i', '--issues', action='store_true',
+        help="Move Open Issues with a milestone to the next milestone.")
+    release_prep_parser.add_argument(
+        '-p', '--prs', action='store_true',
+        help="Move Open PRs with a milestone to the new milestone.")
+
+    release_parser = subparsers.add_parser(
+        'release', help="Tasks to be done during the stabilization phase.")
+    release_parser.add_argument(
+        '-t', '--tag', action='store_true',
+        help="Show commands to properly tag the branch and start the release.")
+    release_parser.add_argument(
+        '-m', '--message', action='store_true',
+        help="Prepare a message to communicate the stabilization process has started.")
+    release_parser.add_argument(
+        '-b', '--bump-version', action='store_true',
+        help="Bump the project version in *master* branch.")
+
+    finish_parser = subparsers.add_parser(
+        'finish', help="Tasks to be done when the release is out.")
+    finish_parser.add_argument(
+        '-m', '--message', action='store_true',
+        help="Prepare a message to communicate the release is out.")
+    return parser.parse_args()
+
+
+def main():
+    args = parse_arguments()
+    ghs = create_github_session(args)
+
+    try:
+        repo = get_repo_object(ghs, args.repository)
+    except Exception as e:
+        print(f'Error: {e}')
+        exit(1)
+
+    if args.subcmd == "stats":
```

Comment:
I propose to solve this by making the functions the same type, i.e. accepting `(repo, args)` + not caring about return value, and to put those in a dictionary, so you get rid of those ifs. Then, you can call `procedures[args.subcmd](repo, args)` inside a new try/except block.

Reply:
Interesting. It should be better. I will take a look. Thanks.

Reply:
Done.